lib/redirect_filter.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const redirectLocation = (redirectTarget, uri) => {
```

Comment:
This ain't gonna work... The problem is that we will receive the uri as internal to the cluster, e.g. `http://restbase.svc.eqiad.wmnet/en.wikipedia.org` etc. 

So your location will also become internal, and the client that receives this redirect will have no idea what to do.

So we need to generate absolute redirect for external requests, and relative redirects for internal requests. normalize_title_filter.js L73-96 does this properly. Now that we have a need for to generate absolute redirects in 2 places, you can extract some code from normalizeTitleFilter into mwutil and use it here too. not sure what the API should be, something like mwUtil.createAbsoluteRedirect